src/sbmlutils/report/sbmlreport.py

```diff
@@ -39,6 +39,7 @@ def create_reports(
     promote: bool = False,
     validate: bool = True,
     math_type: str = "cmathml",
+    report_variable: bool = False,
```

Comment:
Do not add a variable. Just let the function always return the report string. You can add the functionality of returning the string without any additional arguments.
Annotate return type and update description.

Reply:
This function must return a list of report strings, i.e. `List[str]`